doc/release/1.13.0-notes.rst

```diff
@@ -100,6 +102,14 @@ invokes``ndarray.__getslice__`` (e.g. through ``super(...).__getslice__``) will
 now issue a DeprecationWarning - ``.__getitem__(slice(start, end))`` should be
 used instead.
 
+``ndarray.conjugate`` no longer returns a view of real data
+-----------------------------------------------------------
```

Comment:
Nitpick - belongs in previous commit, but I don't think that's worth your time to fix